### DISCUSSION TITLE
pdserving multi-model deployment

### DISCUSSION THREAD

**Discussion by vinnyt123** [PaddlePaddle/PaddleOCR#6840] [Q&A]
I'm wondering if it is possible to deploy multiple language models at once with pdserving?

Ideally I would like to have multiple recognition inference models available so I can conditionally do recognition based on a language parameter.

I see the inference model location is defined in in the [config.yml](https://github.com/vinnyt123/PaddleOCR/blob/dd3214b621b7c07ab67fd51ae2bd9276aa0bbdea/deploy/pdserving/config.yml#L63). Is it possible to change this location at runtime? I am using the python mode for deployment.

**Comment by tink2123** [comment]
There is no direct parameter to support the above behavior, but you can try to implement it like this: 
1. start multiple server services 
2. judge the distribution by customizing the language parameters on the client side.
